### DISCUSSION TITLE
Can we disable AzuraCast adding ?_ic2=1708938087070 to icecast

### DISCUSSION THREAD

**Discussion by redimongo** [AzuraCast/AzuraCast#6974] [General]
I am having issues with feeding the livestream icecast to our own icecast recorder software as Azuracast seems to add `_ic2` to the end of every icecast connection.

I would like to disable this feature please.

Should not a few radio stations are having issues with this causing issues in iOS as it acts as a redirection.

**Comment by mdrxy** [comment]
Facing the same issue here after fiddling with NGINX redirects/custom locations.

**Comment by IKHMedia** [comment]
I found this code that works. 

Here's what I did
nano /etc/nginx/conf.d/ice.conf 

I added the following code to it

#remove query string/s
if ($args ~* "(utm_|gclid|fbclid|_ic2)") {
 #? in uri? drops the utm query string
 rewrite ^(.*)$ $uri? permanent;
}

it works for any query string you can come up with
